README.md

```diff
@@ -163,62 +146,42 @@ To run the project, execute the following command:
 To run tests, run the following command:
 
 ```sh
-  bin/rails rspec spec
-```
-
-### Deployment
-
-You can deploy this project using:
-
-<!--
-Example:
-
-```sh
-
+  rspec spec
```

Comment:
- [ ] This sole command won't do for all the tests:

![image](https://github.com/soulemanou-software/luxury-cars-backend/assets/57421823/e756036b-8fac-40f6-a55d-3b0cf8e6526b)

## Explanation

This is because swagger tests are getting in the way.

You'll need to be more specific on your instructions:

```bash
rspec ./spec/models
rspec ./spec/
rspec ./spec/user/request
```

That way, only rspec tests will be run, and swagger won't get in the way. 👍 

- [ ] You have some failed tests:

### ./spec/models/reservation_spec.rb

![image](https://github.com/soulemanou-software/luxury-cars-backend/assets/57421823/68d9097c-281e-44f1-b55f-2d1396819c20)

All the reservation tests are failing because of this:

![image](https://github.com/soulemanou-software/luxury-cars-backend/assets/57421823/5d5c780e-4073-4ed2-ac71-ae4d381c5984)

... but this project is not about houses.

I changed that variable to `@car` and it worked.

### ./spec/models/user_spec.rb

The last test is failing:

![image](https://github.com/soulemanou-software/luxury-cars-backend/assets/57421823/43ce4dfd-ee5d-4f53-b32e-40e1dc51612f)

Again, it seems to be a problem about houses.  But you are renting cars:

![image](https://github.com/soulemanou-software/luxury-cars-backend/assets/57421823/7e6b6a99-3690-4615-a2f7-88a8a2e65d5d)

I changed the word "houses" to "cars" and it worked.

###  ./spec/user/request

The fourth test is failing:

![image](https://github.com/soulemanou-software/luxury-cars-backend/assets/57421823/f6221591-083f-4e5b-91c1-d40b67b448c2)

This is the reason:

![image](https://github.com/soulemanou-software/luxury-cars-backend/assets/57421823/b7d4b049-737b-497a-a1c4-a3e13256046f)

The expected message and the dealt message from `response.body` are different.



Now that all tests have been debugged, kindly correct them so they all pass. 💪